src/main/scala/viper/gobra/frontend/info/implementation/property/ConstantEvaluation.scala

```diff
@@ -11,6 +11,7 @@ import viper.gobra.frontend.info.base.SymbolTable.SingleConstant
 import viper.gobra.frontend.info.base.Type.{BooleanT, IntT}
 import viper.gobra.frontend.info.implementation.TypeInfoImpl
 import viper.gobra.util.TypeBounds._
+import viper.gobra.util.Violation
```

Comment:
I do not think that his is much of an issue and just a matter of preference. I tend to use `Violation.violation` because I see it as a function of an object, but I do not mind if people use and import and then only call `violation`.

Reply:
this is a very small point and not worth discussing too much. I just think that here, we could be consistent.

Reply:
Ah, I missed that this is the file I changed because it caused a warning. I will change it.